Providers/nxOMSAutomationWorker/automationworker/worker/httpclient.py

```diff
@@ -114,18 +114,44 @@ def delete(self, url, headers=None, data=None):
         """
         pass
 
-
 class RequestResponse:
     """Encapsulates all request response for http clients. Will also deserialize the response when the raw response
     data is deserializable.
     """
+    @staticmethod
+    def check_if_service_unavailable_response_is_received(response_body):
+        SERVICE_UNAVAILABLE_STR = 'Service Unavailable'
```

Comment:
```suggestion
        SERVICE_UNAVAILABLE_STR = 'Service Unavailable'
         HTML_HEADER_TAG_REGEX = '<h[0-9]+>(.*?)</h[0-9]+>'
        HTML_BODY_TAG_REGEX = '<BODY>(.*?)</BODY>'
        HTML_PARAGRAPH_TAG_REGEX = '<p>(.*?)</p>'
```

Can we move it outside or somewhere else. this will create objects everytime the method is run.